assets/batch_score/components/driver/src/batch_score_oss/common/configuration/configuration.py

```diff
@@ -100,25 +98,16 @@ def log(self):
 
     def is_embeddings(self) -> bool:
         """Check if the target endpoint is for embeddings models."""
-        return (
-            self.request_path == constants.DV_EMBEDDINGS_API_PATH
-            or (
-                self.scoring_url is not None
-                and (
-                    self.scoring_url.endswith(constants.DV_EMBEDDINGS_API_PATH)
-                    or self.api_type in [ApiType.Embedding, "embeddings"]
-                )
-            )
-        )
+        return self.api_type in [ApiType.Embedding, "embeddings"]
```

Comment:
Would you mind elaborating why we are removing existing validations?

Reply:
embedding would not be supported post my change for oss. I am just keeping minimal changes related to embedding to not make an already big change in this PR more confusing. Will remove embedding related changes in the future PRs